RELEASE.md

```diff
@@ -1,3 +1,29 @@
+# Release 0.1.0
+
+## New features
+- Telemetry. Marqo now includes various timing metrics for many API endpoints when 
+the query parameter `telemetry=True` is specified (https://github.com/marqo-ai/marqo/pull/506).
+- Consolidate default device to CUDA when available (https://github.com/marqo-ai/marqo/pull/508).
+- Model download integrity verification (https://github.com/marqo-ai/marqo/pull/502).
+
+## Breaking changes
+- Remove deprecated `add_or_update_documents` endpoint (https://github.com/marqo-ai/marqo/pull/517).
+- Disable automatic index creation. Marqo will no longer automatically create an index if it does not exist 
+(https://github.com/marqo-ai/marqo/pull/516).
```

Comment:
A benefit is more certainty about the properties of an index that you add_documents to